lua/ulx/modules/sh/cfc_crucify.lua

```diff
@@ -0,0 +1,146 @@
+CFCUlxCommands.crucify = CFCUlxCommands.crucify or {}
+local cmd = CFCUlxCommands.crucify
+
+CATEGORY_NAME = "Fun"
+
+function cmd.uncrucifyPlayer( ply )
+    if not IsValid( ply ) then return "Invalid player!" end
+
+    local cross = ply.cross
+    ply.cross = nil
+
+    local ragdoll = ply.ragdoll
+    if IsValid( ragdoll ) then
+        ulx.unragdollPlayer( ply )
+    end
+
+    if IsValid( cross ) then
+        cross:Remove()
+    end
+end
+
+local function crucifyPlayer( ply )
+    if not IsValid( ply ) then return "Invalid player!" end
+
+    ulx.ragdollPlayer( ply )
+    local playerRagdoll = ply.ragdoll
+
+    if not IsValid( playerRagdoll ) then return "Ragdoll failed" end
+
+    local cross = ents.Create( "prop_physics" )
+    cross:SetModel( "models/props_c17/gravestone_cross001a.mdl" )
+    cross:SetPos( ply:GetPos() )
+
+    local angles = ply:GetAngles()
+    cross:SetAngles( angles )
+
+    cross:Spawn()
+    cross:Activate()
+
+    local crossPhys = cross:GetPhysicsObject()
+    crossPhys:SetMass( 1000 ) -- make it easier to move the ragdoll attached to the cross
+    crossPhys:EnableMotion( false )
+
+    ply.cross = cross
+    cross.crucifiedPly = ply
+    playerRagdoll.crucifiedPly = ply
+
+    local phys_obj = playerRagdoll:GetPhysicsObjectNum( 1 )
+    local boneID = 2
+
+    while IsValid( phys_obj ) do
+        phys_obj = playerRagdoll:GetPhysicsObjectNum( boneID )
+
+        if phys_obj then
+            phys_obj:SetPos( phys_obj:GetPos() + Vector( 0, 0, 80 ) + angles:Forward() * 10 )
+        end
+        boneID = boneID + 1
+    end
+
+    local rightHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_R_Hand" )
+    local leftHandID = playerRagdoll:LookupBone( "ValveBiped.Bip01_L_Hand" )
+
+    local rightHandPhys = playerRagdoll:TranslateBoneToPhysBone( rightHandID )
+    local leftHandPhys = playerRagdoll:TranslateBoneToPhysBone( leftHandID )

```

Comment:
Same for these, make sure to account for cases where it doesnt return what you want
![image](https://user-images.githubusercontent.com/69946827/231762308-23ece3d8-c478-438a-a441-bd4d6ed8935a.png)